modules/context/repo.go

```diff
@@ -231,9 +231,9 @@ func (r *Repository) FileExists(path, branch string) (bool, error) {
 
 // GetEditorconfig returns the .editorconfig definition if found in the
 // HEAD of the default repo branch.
-func (r *Repository) GetEditorconfig(optCommit ...*git.Commit) (*editorconfig.Editorconfig, error) {
+func (r *Repository) GetEditorconfig(optCommit ...*git.Commit) (*editorconfig.Editorconfig, error, error) {
```

Comment:
Probably helpful to stick names on these return options so it's clearer that the 1st error is actually a warning and the second one is the real error. e.g.

```suggestion
func (r *Repository) GetEditorconfig(optCommit ...*git.Commit) (cfg *editorconfig.Editorconfig, warning error, err error) {
```